services/api/db/serializers.py

```diff
@@ -0,0 +1,132 @@
+from rest_framework import serializers
+from db.models import User
+from django.utils.encoding import smart_str, force_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+#Method for validating password.
+def validate_confirm_password(password,confirm_password):
+    if password != confirm_password:
+        return False
+    return True
+
+class SignUpEndPointSerializer(serializers.ModelSerializer):
+    confirm_password = serializers.CharField(style={'input_type': 'password'}, write_only=True)
+    class Meta:
+        model = User
+        fields= ['email', 'name', 'password', 'confirm_password']
+        extra_kwargs={
+            'password': {'write_only': True}
+        }
+    #To validate whether the entered email and password are correct.
+    def validate(self, data):
+        email = data.get('email')
+        if not email:
+            raise ValueError("Email field is required")
+        password, confirm_password = data.get('password'), data.get('confirm_password')
+        if not password and not confirm_password:
```

Comment:
```suggestion
        if not password or not confirm_password:
```
should the exception be raised when either of the password is not there?

Reply:
if any one them will not there then it will throw error (password and confirm password doesn't match), that's why i didn't write "OR". should i do it?

Reply:
chnaged it to OR"